### ISSUE TITLE
File retains references/indexes to deleted or moved files.

### ISSUE THREAD

**Issue by ranjantomtom** [OpenGrok/OpenGrok#1298]
Using OpenGrok, can you please confirm if it retains references/indexes forndeleted or moved files. 

example, searching for MapData.hpp, the results contain //depot-navkit/dev-michi-curry/Engines/VisualisationInfo/Environment/NavKit/Implementation/MapData.hpp which has been deleted in perforce.

**Comment by vladak** [comment]
Deleted/moved files should be removed from index upon reindex.

**Comment by ranjantomtom** [comment]
We sync and index OpenGrog on daily basis through cronjob but still files not get updted.

**Comment by vladak** [comment]
Just to clarify - the files are deleted on disk, correct ?

**Comment by ranjantomtom** [comment]
Yes, files are modified on the other system we just sync it to opengrok.
